On The Map/BorderedButton.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  BorderedButton.swift
+//  On The Map
+//
+//  Created by Avinash Mudivedu on 4/6/16.
+//  Copyright © 2016 Avinash Mudivedu. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - BorderedButton: Button
+
+class BorderedButton: UIButton {
```

Comment:
Nit: naming convention is to have a three lettered prefix, like GCD and OTM in other files in this project. Consider adding it here.